tutorials/polkadot-sdk/build-a-blockchain/simulate-a-network.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Simulate a Network
+description: Create a private blockchain network with authorized validators using Polkadot-SDK. Start nodes, connect peers and verify block production.
+---
+
+# Simulate a Network
+
+## Introduction
+
+This tutorial introduces you to the process of initiating a private blockchain network with a set of authorized validators. The [Polkadot-SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank} implements an authority consensus model to regulate block production.
+
+In this model, the creation of blocks is restricted to a predefined list of authorized accounts, known as "authorities", who operate in a round-robin fashion. 
```

Comment:
```suggestion
In this model, the creation of blocks is restricted to a predefined list of authorized accounts, known as "authorities," who operate in a round-robin fashion. 
```
This actually is true about the comma being inside the quotation marks. I know we keep it on the outside for things in backticks but, if this is in quotes the comma goes on the inside.